---
layout: page
title: Who is ready to make some science?
---
~ *Cave Johnson*

<hr color = '#fff'>

### Research

Development of statistical methods for analysis of phylogenetics or other systems where individuals share a common origin. The applications of stochastic simulations to evolutionary theory. Namely, multilocus simulaiton of sexually antagonistic alleles and mutation. Statistical phylogenetic analysis of discrete and continuous trait evolution. 

<hr color = '#fff'>

### Publications

1. Armstrong A, N. W. Anderson, H. Blackmon. Inferring the potentially complex genetic architectures of adaptation, sexual dimorphism, and genotype by environment interactions by partitioning of mean phenotypes. Journal of Evolutionary Biology. 2019; 32:4 369-379. <https://doi.org/10.1111/jeb.13421>

    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Genetic architecture fundamentally affects the way that traits evolve. However, the mapping of genotype to phenotype includes complex interactions with the environment or even the sex of an organism that can modulate the expressed phenotype. Line cross analysis is a powerful quantitative genetics method to infer genetic architecture by analyzing the mean phenotype value of two diverged strains and a series of subsequent crosses and backcrosses. However, it has been difficult to account for complex interactions with the environment or sex within this framework. We have developed extensions to line cross analysis that allow for gene by environment and gene by sex interactions. Using extensive simulations studies and reanalysis of empirical data, we show that our approach can account for both unintended environmental variation when crosses cannot be reared in a common garden and can be used to test for the presence of gene by environment or gene by sex interactions. In analyses that fail to account for environmental variation between crosses we find that line cross analysis has low power and high false positive rates. However, we illustrate that accounting for environmental variation allows for the inference of adaptive divergence, and that accounting for sex differences in phenotypes allows practitioners to infer the genetic architecture of sexual dimorphism.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[R documentation](https://rdrr.io/cran/SAGA/)

<hr color = '#fff'>

### Current Projects


1. **How much water is in the fountain of youth?** 

    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Among species that have separate sexes, sex chromosomes are nearly ubiquitous and yet there are many unanswered question with regard to their evolutionary dynamics. XY sex chromosome systems are one of the most common methods of sex determination and have long interested researchers. Normally X and Y chromosomes differentiate over time as the Y chromosome decays. However, not all species experience this Y decay. The fountain of youth hypothesis suggests that imperfect sexual development and deleterious mutations on Y chromosomes may act together as a force to maintain homology between the X and Y. However, the viability of the fountain of youth hypothesis has not been well explored mathematically. Using both finite and infinite population genetic models we have shown that this process cannot completely eliminate sexually antagonistic selection â€“ the force that is thought to lead to the decay of Y chromosomes. Using our modeling approach we are able to determine the parameter space under which the fountain of youth can and cannot preserve similarity between the X and Y chromosome. These results appear to support fountain of youth hypothesis by showing the limits to the canonical model of sex chromosome evolution and grant insight into the fitness effect of sex chromosome inversions.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Manuscript in preparation](https://docs.google.com/document/d/1mfFIwkd4EN3QPzfqcI2Nm-oDbcBi21I5nF7PLdXgnlY/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://drive.google.com/file/d/1z3TgDcqhsfpzkswb43SNzQeYvo4kCNtR/view?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Gitbub.](https://github.com/NW-Anderson/FOY)


2. **Ancestral State Test: a method for inferring the impact of causal relationships between continuous and discrete trait evolution.**

    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Co-evolutionary relationships where the value of a continuous trait affects the evolution of a discrete trait remain difficult to study.  We present a novel comparative method, to be included in the existing R package evobiR, that tests whether a discrete trait transitions when a continuous trait has values more extreme than expected if both traits were evolving independently.  We show that this approach has acceptable type 1 error rates, even when its assumptions are violated, but has low power unless the number of taxa is large. Finally we compare this approach to existing methods that can be used to explore similar hypotheses.  The types of questions that this approach allows us to test are common in evolutionary biology and, unlike existing approaches, the ancestral condition test uses all available data and provides more information than existing methods.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Manuscript in preparation](https://drive.google.com/file/d/1qbt7xyl8Zx57zZjByoCz_Q16L7DEoZLj/view?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/Ancestral-Condition-Test).&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

3. **The Temporal Contingency Test: discovering correlation in the evolution of discrete traits.**

    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;One of the central questions of evolutionary biology is how traits interact with each other over the course of evolution.  However, most of the methods available for understanding correlation or contingency in the evolution of discrete traits are based on detecting differences in the rate of transitions in one trait when it is associated with a specific state of the other trait.  These methods lead to several known problems.  We solve these problems by developing an approach that focuses on temporal contingencies in the transitions of discrete traits.  Our statistical approach can determine whether transitions in one trait lead to transitions in a second trait more quickly than would be expected under a model where the two traits evolve independently.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://docs.google.com/presentation/d/1pk9tAbmJ4eOlU8Y5A-5d2FPTz_BxWOH-tNLFeFKgdCU/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Talk](https://docs.google.com/presentation/d/1Pwcj7orPf9Db11KuQuzQ_T8qJb2X_6rngYaSdsLDUtE/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/TempCorr).

<hr color = '#fff'>

### [CV](https://docs.google.com/document/d/1x__x_N1p2K2cdQtj4fG9xxhNSBCTvy_BpAZppW_HBHY/edit?usp=sharing)


